src/main/java/com/duke/util/Ui.java

```diff
@@ -0,0 +1,62 @@
+package com.duke.util;
+
+import java.util.Scanner;
+
+/**
+ * A user interface that can show message.
+ */
+public class Ui {
+
+    private final String LOGO;
+    private final String LINE;
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public Ui() {
+        this.LOGO = "\t ____        _        \n"
+                + "\t|  _ \\ _   _| | _____ \n"
+                + "\t| | | | | | | |/ / _ \\\n"
+                + "\t| |_| | |_| |   <  __/\n"
+                + "\t|____/ \\__,_|_|\\_\\___|\n";
+        this.LINE = "\t____________________________________________________________________";
+    }
+
+    /**
+     * Show the welcome message when Duke is run.
+     */
+    public void showWelcome() {
```

Comment:
I like how your method names are descriptive and easy to understand the method's purpose! 👍